### ISSUE TITLE
NPE due to nullable customBindings in DynamicMappings

### ISSUE THREAD

**Issue by anallely** [gextech/binding-util#20]
When declaring a dynamic mapping: Exclusions without dynamic binding provokes a null pointer exception.

Example:

```groovy
new DynamicMapping(
        sourceClass: gex.serling.pub.dto.v1.CatalogValue,
        destinationClass: gex.serling.pub.domain.CatalogValue,
        exclusions: ['catalog', 'tenant', 'parent']
      ),
```

Workaround (adding empty customBindings)  :

```groovy
new DynamicMapping(
        sourceClass: gex.serling.pub.dto.v1.CatalogValue,
        destinationClass: gex.serling.pub.domain.CatalogValue,
        customBindings: ["property" : {x -> x}],
        exclusions: ['catalog', 'tenant', 'parent']
      ),
```

**Comment by anallely** [comment]
It would be solved on 0.1.16

**Comment by anallely** [comment]
It was fixed on version #24  (0.1.16)
